taichi/transforms/demote_operations.cpp

```diff
@@ -73,6 +73,29 @@ class DemoteOperations : public BasicStmtVisitor {
         modifier.insert_before(stmt, std::move(floor));
         modifier.erase(stmt);
       }
+    } else if (stmt->op_type == BinaryOpType::bit_shr &&
+               is_integral(lhs->element_type()) &&
+               is_integral(rhs->element_type()) &&
+               is_signed(lhs->element_type())) {
+      // @ti.func
+      // def bit_shr(a, b):
+      //     signed_a = ti.cast(a, ti.uXX)
+      //     shifted = ti.bit_sar(a, b)
+      //     ret = ti.cast(a, ti.iXX)
```

Comment:
According to my understanding, these lines are the explanation of how we demote the operator. If so, maybe use ti.bit_sar(signed_a, b) and ti.cast(shifted, ti.iXX) here is better.
```suggestion
      //     shifted = ti.bit_sar(signed_a, b)
      //     ret = ti.cast(shifted, ti.iXX)
```